app/models/event_stream.rb

```diff
@@ -24,4 +24,12 @@ class EventStream < ApplicationRecord
   belongs_to :container_node
 
   belongs_to :middleware_server, :foreign_key => :middleware_server_id
+
+  after_commit :emit_notifications, :on => :create
```

Comment:
@isimluk @gmcculloug discussed this and we're afraid that this callback could cause a bottle neck in event process given the high frequency of incoming events. It will require a lookup of the event_type for every event `NotificationType.find_by_name(event.event_type)`. 

I think the simplest way around this would be to cache all the instances of `NotificationType` since those will not be changing, afaik, during runtime.